README.md

```diff
@@ -1,18 +1,18 @@
-<h1>Machine Learning coursework description</h1>
-
-<h6>The following to be included:</h6>
-
-<ol>
-  <li>Matlab code</li>
-  <liProblem description</li>
-<li>Initial analysis of the data set including basic statistics</li>
-<li>ML models pros and cons</li>
-<li>Hypothesis statement</li>
-<li>Description of choice of training and evaluation methodology</li>
-<li>Choice of parameters and experimental results</li>
-<li>Analysis and critical evaluation of results</li>
-<li>Lessons learned and future work </li>
-<li>Reference list</li>
-  </ol>
-  
-  <p><b>The list is subject to change<b></p>
+<h1>Machine Learning coursework description</h1>

```

Comment:
Shouldn't be the case anymore, but if you make any future push, please don't push the csv files. it will result in conflicts (time-consuming ones).